File: CS/FilteredData/MyGridControl/MyGridControl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.Grid;

namespace FilteredData.MyGridControl
{
    public class MyGridControl : GridControl
    {

        public MyGridControl()
        {
        }

        protected override DataViewBase CreateDefaultView()
        {
            return new MyTreeListView();
        }
    }
}

File: VB/FilteredData/MyGridControl/MyGridControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.Grid

Namespace FilteredData.MyGridControl
	Public Class MyGridControl
		Inherits GridControl

		Public Sub New()
		End Sub

		Protected Overrides Function CreateDefaultView() As DataViewBase
			Return New MyTreeListView()
		End Function
	End Class
End Namespace
